challenge_presentation/src/challenge_presentation/presentation.py

```diff
@@ -52,6 +52,13 @@ def __init__(self, robot, language='en'):
         smach.State.__init__(self, outcomes=["done", "preempted"])
 
         self.robot = robot
+
+        left_trajectories = ["wave_front", "show_gripper", "point_to_laser"]
+        self.left_arm = self.robot.get_robot_arm(gripper_type='yes', trajectories=left_trajectories, preferred='left!')
+
+        right_trajectories = ["show_gripper", "point_to_kinect"]
+        self.right_arm = self.robot.get_robot_arm(gripper_type='yes', trajectories=right_trajectories, preferred='right!')
```

Comment:
Gripper type doesn't matter. I'd say make gripper_type default to None and let that mean 'any type of gripper will do'